### ISSUE TITLE
Add doc to StratifiedShuffleGroupSplit

### ISSUE THREAD

**Issue by hyruuk** [arthurdehgan/NeuroPy-MLToolbox#1]
I keep getting confused by the proper usage of StratifiedShuffleGroupSplit, and the context in which I should use it instead of sklearn's SSS and GSS.
The doc should specify that this function must be used in case of subject-wise (or group-wise) classification, where all the trials of one subject/group have the same label. 

Additionally, the function should ask for n_splits (like in https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupShuffleSplit.html) instead of n_groups.
